### PR TITLE
W-13672444-excel-controlled-loop-cv

### PR DESCRIPTION
**Pull request by Cristian-Venticinque**

### CODE REVIEW

File: rpa-builder/6.4/modules/ROOT/pages/toolbox-excel-operations-excel-controlled-loop.adoc [hunk 1]
```diff
@@ -65,92 +58,87 @@ include::partial$toolbox-excel-controlled-loop.adoc[tag=troubleshootingSettings]
...
+The row to start reading from in the Excel sheet.
 * *Regex to match*
 
-//link:\l[*Regex condition*]
-
 == Outbound Variables
 
-The contents of each column are returned as variables within the _Excel
-Controlled Loop Action Step_.
+Excel Controlled Loop returns the contents of each column are returned as variables.
```
Comment (ms-tlu) [inline rpa-builder/6.4/modules/ROOT/pages/toolbox-excel-operations-excel-controlled-loop.adoc]: Typo: one "return" too many ;)

**Review by ms-tlu — DISMISSED** [review]
One typo, otherwise LGTM :)

File: rpa-builder/6.4/modules/ROOT/pages/toolbox-excel-operations-excel-controlled-loop.adoc [hunk 2]
```diff
@@ -1,62 +1,55 @@
...
 xref:toolbox-flow-control-file-controlled-loop.adoc[File Controlled Loop] and also forms the
-framework for other Action Steps. The wizard provides you with numerous
-setting options for determining the start (Read mode) and the end
+framework for other Action Steps. The wizard provides setting options for determining the start (Read mode) and the end
 (Terminate loop) of the iteration.
 
-You can only use the _Excel Controlled Loop_ Action Step within an Excel
-session. You enter the source file in the Excel Session.
+The _Excel Controlled Loop_ Action Step works only within an Excel
+session, in which you specify the file to read.
```
Comment (glenn-rodgers-sf) [inline rpa-builder/6.4/modules/ROOT/pages/toolbox-excel-operations-excel-controlled-loop.adoc]:
```suggestion
Session, in which you specify the file to read.

```

File: rpa-builder/6.4/modules/ROOT/pages/toolbox-excel-operations-excel-controlled-loop.adoc [hunk 3]
```diff
@@ -1,62 +1,55 @@
...
+
+Excel Control Loop automatically recognizes the data type of each column. For Date types, the Action Step uses the system's locale settings to read and display the dates, ensuring that the format remains unchanged.
```
Comment (glenn-rodgers-sf) [inline rpa-builder/6.4/modules/ROOT/pages/toolbox-excel-operations-excel-controlled-loop.adoc]:
```suggestion
Excel Control Loop automatically recognizes the data type of each column. For Date types, the Action Step uses the system's locale settings to read and display the dates to ensure that the format remains unchanged. 

```

**Pull request merged**